scl/cisco/plugin.conf

```diff
@@ -72,6 +72,7 @@ block parser cisco-triplet-parser(template() prefix()) {
             };
             rewrite { set("${`prefix`facility}-$1" value('`prefix`facility')); };
         };
+	rewrite { set-severity("${`prefix`severity}"); };
```

Comment:
I think this should go to cisco-parser() and not here, the triplet parser is a low level building block that should only change name-value pairs that it is supposed to extract into. It is ok to do this at the cisco-parser() level, but I don't think it would be ok here.

Reply:
Fixed!